### ISSUE TITLE
Export incomplete AA / nt sequences

### ISSUE THREAD

**Issue by dbolotin** [milaboratory/mixcr#389]
E.g.:

`-nFeatureIncomplete` / `-aaFeatureIncomplete`:

I see two possible ways of how it can be implemented:

1. Export lower-case germline sequences for the parts not covered by original data:

    `-nFeatureIncomplete FR3`:
    ```
    ccagacaaatcagggctgcccagtgatcgcttctctgcagagaggactGGGGGATCCGTCTCCACTCTGACGATCCAGCGCACACAGCAGGAGGACTCGGCCGTGTATCTCTGT
    ```
    
    `-aaFeatureIncomplete FR3`:
    ```
    pdksglpsdrfsaertGGSVSTLTIQRTQQEDSAVYL
    ```

2. Export placeholders for the data not originating from the raw data:

    `-nFeatureIncomplete FR3`:
    ```
    ................................................GGGGGATCCGTCTCCACTCTGACGATCCAGCGCACACAGCAGGAGGACTCGGCCGTGTATCTCTGT
    ```
    
    `-aaFeatureIncomplete FR3`:
    ```
    ................GGSVSTLTIQRTQQEDSAVYL
    ```

P.S. This will break backward compatibility, but this options may be renamed to e.g. `-*FeatureIncomplete` -> `-*Feature`; current `-*Feature` -> `-*FeatureComplete`.

**Comment by ot0tot** [comment]
For my workflow option 1 would be ideal. It would also be trivial to remove the lowercase sequence/replace with placeholders if you needed option 2.

**Comment by mikessh** [comment]
I'd rather call this feature:

-nFeatureDeduced / -aaFeatureDdeuce

**Comment by PoslavskySV** [comment]
Closed via #427
